apps/remix-ide/src/app/providers/injected-provider.tsx

```diff
@@ -61,7 +65,12 @@ export abstract class InjectedProvider extends Plugin implements IProvider {
       this.call('notification', 'toast', this.notFound())
       throw new Error(this.notFound())
     } else {
-      this.askPermission(true)
+      try {
+        await this.askPermission(true)
+      } catch (error) {
+        this.call('notification', 'toast', this.notFound())
```

Comment:
The message does not say anything about the connection.
We can prepend the text here.
Something like " Please make sure your Injected Provider is connected to Remix." @Aniket-Engg ?

Reply:
looks fine to me

Reply:
Thanks for the suggestion. I've made the change in the latest commit. Let me know if there's anything else to tweak :)